xarray/core/pycompat.py

```diff
@@ -101,3 +101,37 @@ def is_chunked_array(x) -> bool:
 
 def is_0d_dask_array(x):
     return is_duck_dask_array(x) and is_scalar(x)
+
+
+def to_numpy(data):
```

Comment:
```suggestion
def to_numpy(data) -> np.ndarray:
```
Not sure what data should be... maybe simply Any?